app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -209,7 +209,7 @@
                             android:orientation="vertical">
 
                             <com.infomaniak.mail.views.itemViews.MyKSuiteStorageBanner
```

Comment:
Does this banner also is suitable for the Pro.Free offer ? 
If yes, maybe we should rename it to KSuiteStorageBanner